spec/features/creating_posts_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+feature 'Creating posts' do
+
+  let(:user){build :user}
+
+  def sign_up(user)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for sign_up is too high. [17/15]